src/shared/avl/utils.ts

```diff
@@ -51,6 +52,32 @@ export const getAvlSubscriptions = async (tableName: string) => {
     return avlSubscriptionsSchema.parse(subscriptions);
 };
 
+export const getAvlSubscriptionErrorData = async (tableName: string, feedId: string): Promise<AvlValidationError[]> => {
+    const now = new Date();
+    const past24Hours = new Date(now.getTime() - 24 * 60 * 60 * 1000).toISOString();
```

Comment:
```suggestion
    const now = getDate();
    const past24Hours = now.subtract(24, "hours");
```

then down below, `past24Hours.toISOString()`